ext/spl/spl_fixedarray.c

```diff
@@ -310,8 +310,8 @@ static inline zval *spl_fixedarray_object_read_dimension_helper(spl_fixedarray_o
 	/* we have to return NULL on error here to avoid memleak because of
 	 * ZE duplicating uninitialized_zval_ptr */
 	if (!offset) {
-		zend_throw_exception(spl_ce_RuntimeException, "Index invalid or out of range", 0);
-		return NULL;
+		zend_value_error("Index invalid or out of range");
+		RETURN_THROWS();
```

Comment:
I think it won't work, because `RETURN_THROWS()` can only be used where the `return_value` variable is present (usually in `PHP_FUNCTION`s or `PHP_METHOD`s).

Reply:
Yeah, I should have tried compiling it 🤦‍♂